src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespaceafter/InputNoWhitespaceAfter.java

```diff
@@ -8,7 +8,7 @@
 
 /**
  * Class for testing whitespace issues.
- * error missing author tag
+ * violation missing author tag
```

Comment:
Is this line necessary? There is 11 "missing author tag" in the inputs. Why not just remove them?